phpunit.xml.dist

```diff
@@ -2,18 +2,15 @@
 
 <phpunit backupGlobals="false"
          backupStaticAttributes="false"
-         colors="false"
+         colors="true"
          convertErrorsToExceptions="true"
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
          stopOnFailure="false"
-         syntaxCheck="false"
-         codecoverage="true"
          bootstrap="./vendor/autoload.php"
         >
 
-    <formatter type="clover" usefile="false"/>
```

Comment:
Fairly speaking, I'm not sure why we have this and exactly it does... but any reasons why it was removed by you?

Reply:
It is just for coverage support. I am okey removing it.